tee-worker/ts-tests/common/utils.ts

```diff
@@ -523,6 +529,9 @@ export async function handleIdentityEvents(
             case 'Failed':
                 results.push(events[index].data.detail.toHuman());
                 break;
+            case 'CreateIdentityFailed':
```

Comment:
Maybe can remove `CreateIdentityFailed`, as "Failed" has a standardized parsing format.

Reply:
Ok, but I still need to add `case 'CreateIdentityFailed':`, otherwise it's not matched and no events are returned from `handleIdentityEvents`. Next, if I call `checkErrorDetail` with an empty list of events it gives a false positive result, I don't know if there was an error with the given message or not.